src/Client.Infrastructure/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -5,7 +5,7 @@ public static class ClaimsPrincipalExtensions
         => claimsPrincipal.FindFirstValue(ClaimTypes.Email);
 
     public static string? GetTenant(this ClaimsPrincipal claimsPrincipal)
-        => claimsPrincipal.FindFirstValue(ClaimTypes.Name);
+        => claimsPrincipal.FindFirstValue(ClaimTypes.Name); // Todo: ?? think this is not right
```

Comment:
Yes, has to be `claimsPrincipal.FindFirstValue("Tenant")` or something.

Reply:
should be ok now